src/components/unapprovedFilterImageControls/index.tsx

```diff
@@ -0,0 +1,145 @@
+  return current > moment().endOf('day');
+}
+
+const activityCountLabels: SliderMarks = {
+  0: '0',
+  [MAX_ACTIVITY_COUNT + 1]: `${MAX_ACTIVITY_COUNT}+`,
+};
+
+const neighborhoodOptions = Object.values(Neighborhoods)
+  .sort()
+  .map((value) => {
+    return { label: value, value };
+  });
+
+interface UnapprovedFilterImageFilterControlsProps {
+  filters: ReviewImageFilters;
+  setFilters: React.Dispatch<React.SetStateAction<ReviewImageFilters>>;
+}
+
+const UnapprovedFilterImageControls: React.FC<
+  UnapprovedFilterImageFilterControlsProps
+> = ({ filters, setFilters }) => {
+  const [siteIdOptions, setSiteIdOptions] = useState<
+    { label: string; value: string }[]
+  >([]);
+
+  const [siteIdInvalidOrNone, setSiteIdError] = useState<{
+    status: '' | 'warning' | 'error' | undefined;
+    value: string;
+  }>({ status: '', value: 'Enter a site id' });
+
+  // TODO: verify what Tree ID means
+  return (
+    <Collapse ghost>
+      <Collapse.Panel header="Date Submitted" key="submittedDate">
+        <DatePicker.RangePicker
+          allowEmpty={[true, true]}
+          value={formatDates(filters.submittedStart, filters.submittedEnd)}
+          onChange={(_, dateStrings) =>
+            setFilters({
+              ...filters,
+              submittedStart: dateStrings[0] || null,
+              submittedEnd: dateStrings[1] || null,
+            })
+          }
+          disabledDate={disabledDate}
+        />
+      </Collapse.Panel>
+      <Collapse.Panel header="Neighborhood" key="neighborhood">
+        <AutoCompleteSelect
+          value={filters.neighborhoods}
+          mode="multiple"
+          allowClear
+          placeholder="Enter a neighborhood"
+          onChange={(value: Neighborhoods[]) =>
+            setFilters({ ...filters, neighborhoods: value })
+          }
+          options={neighborhoodOptions}
+        />
+      </Collapse.Panel>
+      <Collapse.Panel header="Tree ID" key="siteId">
+        <AutoCompleteSelect
+          value={filters.siteIds}
+          mode="multiple"
+          allowClear
+          placeholder="Enter a site"
+          onChange={(value: number[]) => {
+            const valueInt = value.map((v) => Number(v) ?? -1);
+            setFilters({ ...filters, siteIds: valueInt });
+          }}
+          status={siteIdInvalidOrNone.status}
+          onSearch={(e) => {
+            const numVal = Number(e);
+            if (numVal) {
+              setSiteIdError({
+                status: '',
+                value: '',
+              });
+            } else if (e === '') {
+              setSiteIdError({
+                status: '',
+                value: 'Enter a site id',
+              });
+            } else {
+              setSiteIdError({
+                status: 'error',
+                value: 'No data. Site IDs must be strings',
+              });
+            }
+            if (e.length > 0) setSiteIdOptions([{ label: e, value: e }]);
+          }}
```

Comment:
Right now, it's possible to select non-numeric text as an option, which gets converted into `NaN` on selection (pic) - can we move the last line in the search function (`if (e.length > 0)...`) into the first `if` branch? I think that should prevent this case from happening by preventing users from selecting an option with mixed numbers and text.

![image](https://github.com/Code-4-Community/speak-for-the-trees-frontend/assets/68914997/4364e0f3-6df3-4c60-8a68-ed5a51cd7622)

Otherwise, the logic and messaging here is awesome - good work!